### PR TITLE
feat(register):New Account Registration

### PR DESCRIPTION
**Pull request by kibo-geetanshuchhabra**

### CODE REVIEW

File: composables/__tests__/useUser.test.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ const loginResponse = {
 
 const createAccountResponse = {
   account: {
-    emailAddress: "firstName@gmail.com",
+    emailAddress: "11",
```
Comment (kibo-kevinwatts) [inline composables/__tests__/useUser.test.ts]: should this be `firstName11@gmail.com`

**Review by kibo-kevinwatts — APPROVED** [review]

File: composables/__tests__/useUser.test.ts (hunk 1 above)
Comment (kibo-geetanshuchhabra) [reply composables/__tests__/useUser.test.ts]: Resolved

File: components/LoginModal.vue [hunk 2]
```diff
@@ -2,12 +2,27 @@
...
       />
+      <div class="smartphone-only">
+        <div class="modal-header">
+          <div class="bar-heading">
+            {{ barTitle }}
+          </div>
+          <SfIcon
+            icon="cross"
+            size="0.875rem"
+            color="#7c7c7c"
```
Comment (kibo-sushant) [inline components/LoginModal.vue]: Instead of hardcoding colors, can we use predefined colors?

File: components/LoginModal.vue [hunk 3]
```diff
@@ -102,49 +120,48 @@
...
-            label="Your email"
+            label="Email*"
             class="form__element"
             type="email"
           />
           <SfInput
             v-model="form.firstName"
             name="first-name"
-            label="First Name"
+            label="First Name*"
```
Comment (kibo-sushant) [inline components/LoginModal.vue]: canwe use localization for email, first name and last name?

**Review by kibo-sushant — APPROVED** [review]

**Pull request merged**